dnf-behave-tests/features/plugins-core/needs-restarting.feature

```diff
@@ -0,0 +1,34 @@
+@bz1639468
+Feature: Reboot hint
+
+Background:
+    Given I enable plugin "needs_restarting"
+      And I use the repository "dnf-ci-fedora"
+
+Scenario: Update core packages
+    Given I execute dnf with args "install kernel glibc"
+      And I use the repository "dnf-ci-fedora-updates"
+      And I execute dnf with args "upgrade"
+     When I execute dnf with args "needs-restarting -r"
+     Then the exit code is 1
+      And stdout contains "  \* kernel"
+      And stdout contains "  \* glibc"
+      And stdout contains "Reboot is required"
+      And stdout contains "https://access.redhat.com/solutions/27943"
+
+Scenario: Update non-core packages
+    Given I execute dnf with args "install wget"
+      And I use the repository "dnf-ci-fedora-updates"
+      And I execute dnf with args "upgrade"
+     When I execute dnf with args "needs-restarting -r"
+     Then the exit code is 0
+      And stdout does not contain "  \* wget"
+      And stdout contains "Reboot should not be necessary"
```

Comment:
Can you check the exact stdout in this case? It should always be just these two lines:
```
No core libraries or services have been updated since boot-up.
Reboot should not be necessary.
```

Reply:
Sure, I'll do.